deploy/020-rate-model-checkpoint.ts

```diff
@@ -0,0 +1,108 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { getConfig } from "../helpers/deploymentConfig";
+import { getRateModelName, getRateModelParams } from "../helpers/rateModelHelpers";
+import { writeGeneratedContract } from "../helpers/writeFile";
+
+const checkpoints = {
+  bsctestnet: {
+    at: 1744097580, // 2025-04-08 07:33:00 AM UTC,
+    fromBlocksPerYear: 10_512_000,
+    toBlocksPerYear: 21_024_000,
+  },
+  bscmainnet: {
+    at: 1745895600, // 2025-04-29 03:00:00 AM UTC
```

Comment:
Should be

```suggestion
    at: 1745903100, // 2025-04-29 05:05:00 AM UTC
```

Reply:
Fixed here (https://github.com/VenusProtocol/isolated-pools/pull/508/commits/79de9e7236bd7bbacb333f3981fcbe1c0af48083) and redeployed